spec/requests/foods_spec.rb

```diff
@@ -0,0 +1,130 @@
+require 'rails_helper'
+
+# This spec was generated by rspec-rails when you ran the scaffold generator.
+# It demonstrates how one might use RSpec to test the controller code that
+# was generated by Rails when you ran the scaffold generator.
+#
+# It assumes that the implementation code is generated by the rails scaffold
+# generator. If you are using any extension libraries to generate different
+# controller code, this generated spec may or may not pass.
+#
+# It only uses APIs available in rails and/or rspec-rails. There are a number
+# of tools you can use to make these specs even more expressive, but we're
+# sticking to rails and rspec-rails APIs to keep things simple and stable.
+
+RSpec.describe '/foods', type: :request do
```

Comment:
- [ ] According to the project requirement, you should have set up RSpec to write Unit tests and  Capybara testing libraries for the Integration tests. Kindly address this 💯 👍🏻  Actually I'm getting an error when I run the test.

Reply:
@BenMukebo according to the [technical setup](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md?plain=1#L128) we only need to install de gems. I did integration tests to challenge myself and use rspec with devise. 
I deleted all the extra files in the spec folder created by scaffold.